pkg/apis/kubeone/types.go

```diff
@@ -167,6 +168,7 @@ type WorkerConfig struct {
 type ProviderSpec struct {
 	CloudProviderSpec   json.RawMessage   `json:"cloudProviderSpec"`
 	Labels              map[string]string `json:"labels"`
+	Taints              []corev1.Taint    `json:"taints"`
```

Comment:
Should we use `corev1.Taints` directly or have our API field and convert to `corev1.Taints` on runtime?

Reply:
In machine spec it’s Taints []corev1.Taint `json:"taints,omitempty"`

Reply:
Done. :+1: